### PR TITLE
Fix code generation failure on zap_generate.sh

### PR DESCRIPTION
**Pull request by markus-becker-tridonic-com**
#### Problem

Error message was:
"Default value for: manufacturerCodes/0x1002 does not match an option."

#### Summary of Changes

manufacturers.xml is in a different location in the source tree. Point to the location in third_party/zap/repo/ within zcl.json.

Signed-off-by: Markus Becker <markus.becker@tridonic.com>

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

**Review by vivien-apple — APPROVED** [review]

**Review by selissia — APPROVED** [review]

File: src/app/zap-templates/zcl/zcl.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
         "sample-extensions.xml"
     ],
 
-    "manufacturersXml": "../shared/manufacturers.xml",
+    "manufacturersXml": "../../../../third_party/zap/repo/zcl-builtin/shared/manufacturers.xml",
```
Comment (andy31415) [inline src/app/zap-templates/zcl/zcl.json:33]: could we instead import the file into CHIP?

I think we are working towards not having source dependency on zap.

@jepenven-silabs ?

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@saurabhst ?

**Review by saurabhst — APPROVED** [review]

File: src/app/zap-templates/zcl/zcl.json (hunk 1 above)
Comment (jepenven-silabs) [reply src/app/zap-templates/zcl/zcl.json:33]: Yes we should move this file within CHIP repo as to no have any ZCL dependency with ZAP. @markus-becker-tridonic-com always assume that if the files within ZAP `zcl-builtin` folder might change at any moment.

Comment (jepenven-silabs) [reply src/app/zap-templates/zcl/zcl.json:33]: @markus-becker-tridonic-com could you copy the `manufacturers.xml` inside the `/src/app/zap-templates/zcl` folder and update the path ?

**Review by jepenven-silabs — CHANGES_REQUESTED** [review]

**Comment by markus-becker-tridonic-com** [comment]
Adapted as requested by @jepenven-silabs and @andy31415.

**Review by jepenven-silabs — APPROVED** [review]

**Review by mspang — APPROVED** [review]

**Pull request merged**